Library/Source/RuntimeImpl.cpp

```diff
@@ -156,6 +156,15 @@ namespace Babylon
                 curl_easy_setopt(curl, CURLOPT_WRITEFUNCTION, callback);
                 curl_easy_setopt(curl, CURLOPT_WRITEDATA, &data);
 
+#if (ANDROID)
+                /*
+                 * /!\ warning! this is a security issue
```

Comment:
Is this going to be a blocking issue in the near future? Should we call this out specifically to interested parties (who might have the expertise to investigate it)?